components/blocks/nav/SiteNavbar/SiteNavbar.test.ts

```diff
@@ -1,34 +1,47 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-expect-error
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import { setup, $fetch, url } from '@nuxt/test-utils-edge'
+import { describe, expect, it, vi } from 'vitest'
+
+// import from 'nuxt'
+
 import { ref } from 'vue'
 import SiteNavbar from './SiteNavbar.vue'
+// import 'root/vitest.setup'
+import { fireEvent, stubbedRender } from 'root/testUtils'
 import * as apiComposables from 'root/composables/api'
 import { User } from 'root/lib/api/data-contracts'
-import { fireEvent, stubbedRender } from 'root/testUtils'
 
-vi.mock('#app', () => ({
-  useState: vi.fn(ref),
+vi.doMock(`${url('#app')}`, () => ({
+  useRuntimeConfig: () => ({
+    public: {
+      BACKEND_BASE_URL: process.env.BACKEND_BASE_URL,
+    },
+  }),
 }))
 
 afterEach(() => {
   vi.restoreAllMocks()
   vi.clearAllMocks()
 })
 
-describe('<SiteNavbar />', () => {
+describe('<SiteNavbar />', async () => {
+  await setup({})
```

Comment:
I take it that we need to call `setup()` at the start of every test suite? Or nah?

Reply:
I think at the beginning of every `describe` but i could be wrong that it only needs it in the top most if a `describe` is nested in others. but for now we'll do this.